README.md

```diff
@@ -160,19 +160,23 @@ Reference:
 Once you have docker running locally for prod, navigate to Ngrok dashboard
 ```
 http://localhost:4040/status
-``` 
-to get the public URL. e.g. `https://05473cbc.ngrok.io` 
+```
+to get the public URL. e.g. `https://05473cbc.ngrok.io`
 
 ### UI MOCK
 
 The UI mock for the project is available [here](https://www.figma.com/file/Yn3JRZ3YLBVSg4o8L9dhIAv2/Andela_Socials)
 
-### DUMMY DATA
+### STATIC DUMMY DATA
```

Comment:
@olusoladavid this section and dynamic dummy data section should be under one single section titled **Database Seeding** for example, and then added as subtitles.

Sample:

```
### Database Seeding
#### Static Dummy Data
#### Dynamic Dummy Data
```

Reply:
Yeah. Sounds better. Thanks for the suggestion. I will update the README with the suggested structure.